backend/pkg/sqlmanager/shared/types.go

```diff
@@ -166,6 +163,9 @@ func (s *SchemaTableDataTypeResponse) GetStatements() []string {
 	for _, domain := range s.Domains {
 		output = append(output, domain.Definition)
 	}
+	for _, fn := range s.Functions {
```

Comment:
Guessing this is to handle the case where functions are using things like enums, domains, etc?

Reply:
exactly